Tools/nib2cib/NSBox.j

```diff
@@ -71,9 +71,8 @@
         }
 
         _title         = [[aCoder decodeObjectForKey:@"NSTitleCell"] objectValue] || @"";
-        _titlePosition = [aCoder decodeObjectForKey:@"NSTitlePosition"];
 
-        if (_titlePosition === undefined)
+        if (_titlePosition === undefined && _boxType !== CPBoxSeparator)
```

Comment:
I somehow convinced myself that it would not work without removing line 74. This left me uneasy. And yet, a quick run just now with line 74 in seems to give the desired result. I need to have a broader look at the class and see what visual/manual tests we have in place. I will try to fit it in over the next day or so.